cmake/ExternalProjects.cmake

```diff
@@ -58,10 +54,42 @@ find_package(Catch2 REQUIRED)
 # https://github.com/aws/aws-sdk-cpp/blob/main/Docs/CMake_External_Project.md
 # but they don't specify how to link the libraries, which required adding an
 # extra couple of CMake targets.
-# We now use the Conan package for aws-sdk-cpp: https://conan.io/center/aws-sdk-cpp
-# To enable only S3, all default targets from the Conan recipe
-# have to be explicitly disabled above
-find_package(AWSSDK REQUIRED)
+# There are some AWS docs on using the cpp sdk as an external project:
```

Comment:
Would be good to put a warning here with a date saying why we're not using Conan for aws stuff in case somebody gets tempted.